evaluation.md

```diff
@@ -5,15 +5,30 @@
 _Describe each change from your ideal syntax to the syntax you implemented, and
 describe_ why _you made the change._
 
+The first change we made was the addition of some parentheses to our statements. This was necessary because we couldn't figure out how to tell scala to give one operator higher precedence than another, so it was simply looking at things from left-to-right which was not always what we wanted.
+
+The second change was removing the "(search)" tag and "go to search" line, and replacing them with a thenLoop statement and a "go (back)" statement. We couldn't think of any way to implement arbitrary tags for lines without putting the loop tag (like "search") in quotes, which would lose a lot of the DSL-y-ness. We needed thenLoop in order to have access to both lines at once, so that we could keep track of where the "go back" statement would need to jump to.
```

Comment:
So the version with 'thenLoop' is equivalent (in terms of power) to the version with 'go to'?  I'm thinking it is, as users can just copy + paste whenever they need to reuse a loop.  This would get a lot wordier, though.